src/azul/health.py

```diff
@@ -270,10 +270,10 @@ def elasticsearch(self):
 
     @lru_cache
     def _lambda(self, lambda_name) -> JSON:
+        url = config.lambda_endpoint(lambda_name).set(path='/health/basic',
+                                                      args={'catalog': self.catalog})
+        response = requests.get(str(url))
```

Comment:
I think the point was for the `.get()` to be inside the `try` block, so that name resolution or connection errors are caught and reported as the endpoint being down.